cmd/run.go

```diff
@@ -62,6 +80,17 @@ var runCmd = &cobra.Command{
 		if err != nil {
 			log.Fatalf("%+v", err)
 		}
+		if env == "prod" {
+			refInfos, err := git.ShowRef(tag)
+			if err != nil {
+				log.Fatalf("%+v", err)
+			}
+			branch = branch[1:len(branch)]
+			fmt.Printf("Will deploy version (%s) with this commit to %s\n------------------------------\n%s\n------------------------------\nContinue?", color.New(color.FgGreen).SprintFunc()(branch), color.New(color.Bold).SprintFunc()("prod"), refInfos)
+		}
+		var ok string
+		fmt.Scanln(&ok)
```

Comment:
You're not checking `ok` result here.

Reply:
thats wanted, you're supposed to ctrl-c to stop or press enter to continue